client/.eslintrc.js

```diff
@@ -13,7 +13,11 @@ module.exports = {
       "skipShapeProps": true
     }],
     "react/no-multi-comp": [0],
-    "no-bitwise": [0],
+    "function-paren-newline": ["error", "consistent"],
```

Comment:
Dont think this is working. I still get hundreds of

Unexpected newline before ')'. (function-paren-newline)

Reply:
wait, ignore me. it should be throwing as an error